### PR TITLE
Code cleanup in serverpackethandler

### PR DESCRIPTION
**Pull request by HybridDog**
* call pkt->getPeerId() only once
* remove redundant vector int to float to int conversion
* some style changes

### CODE REVIEW

**Comment by HybridDog** [comment]
It works for me.

File: src/network/serverpackethandler.cpp [hunk 1]
```diff
@@ -998,32 +998,43 @@ void Server::handleCommand_Interact(NetworkPacket *pkt)
...
 
+	// Get pointed to node, may be undefined
+	v3s16 p_under = pointed.node_undersurface;
+	v3s16 p_above = pointed.node_abovesurface;
+	/* Uncomment when players can place a node onto an object.
+	if (pointed.type == POINTEDTHING_OBJECT) {
+		p_under = floatToInt(pointed_object->getBasePosition(), BS);
+		p_above = p_above;
+	}
+	*/
```
Comment (sfan5) [inline src/network/serverpackethandler.cpp]: I don't think keeping "theoretical support code" like this around is useful.

File: src/network/serverpackethandler.cpp [hunk 2]
```diff
@@ -998,32 +998,43 @@ void Server::handleCommand_Interact(NetworkPacket *pkt)
...
 		errorstream << "Server::ProcessData(): Canceling: "
-				"No player object for peer_id=" << pkt->getPeerId()
+				"No player object for peer_id=" << peer_id
 				<< " disconnecting peer!" << std::endl;
-		DisconnectPeer(pkt->getPeerId());
+		DisconnectPeer(peer_id);
 		return;
 	}
 
+	// Get pointed to node, may be undefined
```
Comment (sfan5) [inline src/network/serverpackethandler.cpp]: and instead of letting this be undefined, why not keep it at the player_pos by default?

**Comment by SmallJoker** [comment]
No response from author in more than a month. Please ask to re-open if there are news.

File: src/network/serverpackethandler.cpp (hunk 2 above)
Comment (HybridDog) [reply src/network/serverpackethandler.cpp]: I've moved this code.

**Comment by HybridDog** [comment]
I've rebased it and added more style changes; please re-open.

**Comment by SmallJoker** [comment]
Sorry, GitHub does not let me re-opening the PR. Weird.. this should be allowed.

grafik

**Comment by HybridDog** [comment]
> Sorry, GitHub does not let me re-opening the PR. 

I've opened a new one.

**Pull request closed without merging**